### DISCUSSION TITLE
Minor version updates?

### DISCUSSION THREAD

**Discussion by aadrian** [chumaky/docker-images#3] [General]
Hi,

Any plans publishing some new updated minor versions?  E.g. :
 - pg 15.2 ->  15.8
 - pg 16.3 -> 16.4
 - mysql_fdw 2.9.1 -> 2.9.2

Thank you.

**Comment by AmebaBrain** [comment]
Hello @aadrian 
Sorry, only now saw this.

Yes, it's possible. But mostly per request. From FDW perspective it's makes more sense only in case there is corresponding FDW version is bumping up. For minor postgres version upgrade there is little to nothing is changed from FDW perspective.  

Please let me know which setup you are interested in. I'll try to make it.

**Reply by aadrian** [reply]
@AmebaBrain  Thank you for your answer

>  it's makes more sense only in case there is corresponding FDW version is bumping up

There is a FDW version bump: 
 -  currently the Docker image has [2.9.1](https://github.com/chumaky/docker-images/blob/6e60e0fcabeeb32b9f6c614d19bc6fb83fa3e6ca/v16/postgres_mysql.docker#L6)
 - but the latest is version 2.9.2  https://github.com/EnterpriseDB/mysql_fdw/releases/tag/REL-2_9_2


> Please let me know which setup you are interested in. I'll try to make it.

I'm using 15 and 16  (the latest), so :
 -   pg 15.2 -> **15.8**
 -   pg 16.3 -> **16.4**
 -  with  mysql_fdw 2.9.1 -> **2.9.2**

and now also started using **PG 17**  https://www.postgresql.org/docs/17/release-17.html

Thanks in advance.

**Reply by AmebaBrain** [reply]
@aadrian 
Created two new tags for `postgres_mysql_fdw` image and pointed `latest` to the 17th [version](https://hub.docker.com/r/chumaky/postgres_mysql_fdw/tags)
```
docker pull chumaky/postgres_mysql_fdw:latest
docker pull chumaky/postgres_mysql_fdw:17.0_fdw2.9.2
docker pull chumaky/postgres_mysql_fdw:16.4_fdw2.9.1
```
Version `2.9.2` is supposed to be used only with 17th version of postgres.

You could check the image by navigating to the `test` folder and spinning up the corresponding compose file
```sh
tests (master) $ docker compose -f postgres_mysql_compose.yml up -d
```
```sql
$ docker exec -it postgres psql postgres postgres

postgres=# select version();
                                                       version                                                       
---------------------------------------------------------------------------------------------------------------------
 PostgreSQL 17.0 (Debian 17.0-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
(1 row)

postgres=# select mysql_fdw.mysql_fdw_version();
 mysql_fdw_version 
-------------------
             20902
(1 row)




postgres=# select version();
                                                       version                                                       
---------------------------------------------------------------------------------------------------------------------
 PostgreSQL 16.4 (Debian 16.4-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
(1 row)

postgres=# select mysql_fdw.mysql_fdw_version();
 mysql_fdw_version 
-------------------
             20901
(1 row)
```
